docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -3,85 +3,46 @@ title: Ingestion Filtering
+
+The feature does not filter the storage of current network ledger state in Horizon database as that is required for referential integrity of entity identity in sql database overall, however db storage for current state is relatively extremely lower than historical concerns and does not increase exponentially, in the order of a few GB's total.
 
-Whitelisting can include the following supported entities:
+Filter rules can whitelist ingestion by the following supported entities:
 
 - Account id
 - Asset id (canonical)
 
-Given that all transactions related to the white listed entities are included, all historical time series data related to those transactions are saved in horizon's history db as well. For example, whitelisting an Asset will also persist all Accounts that interact with that Asset and vice versa, if an Account is whitelisted, all assets that are held by that Account will also be included.
+Given that all transactions related to the white listed entities are included, all historical time series data related to those transactions are saved in horizon's history db, including transaction itself, all operations in the transaction and references to any ancillary entities from operations.
 
 ## Configuration:
 
-The filters and their configuration are optional features and must be enabled with horizon command line or environmental parameters:
-
-```
-admin-port=[your_choice]
-```
-
-and
-
-```
-exp-enable-ingestion-filtering=true
-```
-
-As Environment properties:
-
-```
-ADMIN_PORT=<port>
-```
-
-and
-
-```
-EXP-ENABLE-INGESTION-FILTERING=True
-```
-
-These should be included in addition to the standard ingestion parameters that must be set also to enable the ingestion engine to be running, such as `ingest=true`, etc. Once these flags are included at horizon runtime, filter configurations and their rules are initially empty and the filters are disabled by default. To enable filters, update the configuration settings, refer to the Admin API Docs which are published as Open API 3.0 doc on the Admin Port at `http://localhost:<admin_port>/`. You can paste the contents from that url into any OAPI tool such as [Swagger](https://editor.swagger.io/) which will render a visual explorer of the API endpoints. Follow details and examples for endpoints:
-
-```
-/ingestion/filters/account
-/ingestion/filters/asset
-```
-
-## Operation:
+Filtering is enabled by default but with no filter rules, which effectively means no filtering of ingested data occurs. To start filtering ingestion:
 
-Adding and Removing Entities can be done by submitting PUT requests to the `http://localhost:<admin_port>/` endpoint.
+- enable Horizon admin port with environmental configuration parameter `ADMIN_PORT=XXXXX`, this will allow you to access the port.
+- define filter whitelists. submit Admin HTTP API requests to view and update the filter rules:
```

Comment:
```suggestion
- enable Horizon admin port with the configuration parameter `ADMIN_PORT=XXXXX`, this will allow you to access the port
- define filter allowlists
- submit Admin HTTP API requests to view and update the filter rules
```